client/src/app/openslides-main-module/components/openslides-main/openslides-main.component.ts

```diff
@@ -27,7 +30,8 @@ export class OpenSlidesMainComponent implements OnInit {
         private domSanitizer: DomSanitizer,
         private openslidesStatus: OpenSlidesStatusService,
         private matIconRegistry: MatIconRegistry,
-        private translate: TranslateService
+        private translate: TranslateService,
+        private storageMap: StorageMap
```

Comment:
We have a `StorageService` in the client, which is used basically everywhere, where something needs to be saved/loaded from the local storage.

You should probably use that instead.